CMakeLists.txt

```diff
@@ -455,10 +455,11 @@ if (BUILD_TESTING)
     if(BENCHMARK)
         find_package(benchmark REQUIRED)
```

Comment:
cmake is complaining there is no package configuration for benchmark.  Double-checked #2582 - not clear where this coming from; is there some bootstrapping that needs to be documented/added somewhere?

Reply:
when building s2n you need to pass in the prefix path for google benchmark:
    -DCMAKE_PREFIX_PATH="~/benchmark/install"

Reply:
How I build the s2n library:

cmake . -Bbuild -GNinja -DCMAKE_EXE_LINKER_FLAGS="-lcrypto -lz" -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=1 -DBENCHMARK=1 -DCMAKE_PREFIX_PATH="~/aws-lc/aws-lc-install;~/benchmark/install"